src/cryptonote_core/blockchain.cpp

```diff
@@ -101,8 +102,9 @@ static const struct {
 } testnet_hard_forks[] = {
   // version 7 from the start of the blockchain, inhereted from Monero testnet
   { 7, 1, 0, 1512211236 },
+  { 8, 54867, 0, 1532328024 },
 };
-static const uint64_t testnet_hard_fork_version_1_till = 624633;
+static const uint64_t testnet_hard_fork_version_1_till = 0;
```

Comment:
Technically this should be changed to `testnet_hard_fork_version_7_till = 54866;` (see https://github.com/monero-project/monero/blob/7d2d8055ac1eb524f3917a713b071772ddc89761/src/cryptonote_core/blockchain.cpp#L136)

These variables are only used once in the Hardfork constructor in Blockchain::init() and are assigned to hardfork.original_version_till_height which is used only in HardFork::get_block_version() and actually looks redundant.

```
uint8_t HardFork::get_block_version(uint64_t height) const
{
  if (height <= original_version_till_height)
    return original_version;

  const cryptonote::block &block = db.get_block_from_height(height);
  return ::get_block_version(block);
}
```

Since blocks should have the right version associated them. But in preference of leaving no room for error I'd just rename and update the values.